README.md

```diff
@@ -46,10 +46,10 @@ CLIENT_ORIGIN_URL=http://localhost:3000
 REACT_APP_API_SERVER_URL=http://localhost:6060
 ```
 
-Then run the following commands to set up your python virtual environment:
+Then run the following commands to set up your python virtual environment. Access can be run with Python 3.10 and above:
 
 ```
-python3.11 -m venv venv
```

Comment:
why remove this? what if the host has installed 2 different python

Reply:
Removing this so we don't forget to update it for every new version of released Python (as what happened here). If someone has multiple python versions supported they can specify the desired version using a similar alias or an absolute path. Ideally, `python3` will point to a recent install if Python >= 3.10.